modules/api/src/main/java/org/apache/ignite/table/manager/IgniteTables.java

```diff
@@ -110,14 +83,15 @@
      *                         <ul>
      *                             <li>the node is stopping.</li>
      *                         </ul>
+     * @see TableNotFoundException
      */
     CompletableFuture<Void> alterTableAsync(String name, Consumer<TableChange> tableChange);
 
     /**
-     * Drops a table with the name specified. If a table with the specified name does not exist in the cluster, the operation has no
-     * effect.
+     * Drops a table with the name specified. If appropriate table does not be found, an exception will be thrown.
```

Comment:
```suggestion
     * Drops a table with the name specified. If appropriate table cannot not be found, an exception will be thrown.
```

Reply:
If an appropriate table does not exist....